### PR TITLE
add axios interceptors for error handling. Add error page

### PR DESCRIPTION
**Pull request by sunitakawane**

### CODE REVIEW

File: src/constants/url.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const url = {
 
-  BASE_API_URL_SERVER: "https://tapestry-pooling-284109.ew.r.appspot.com/",
+  BASE_FRONTEND_URL: "http://localhost:3000/",
```
Comment (Dheeraj-DJP) [inline src/constants/url.js:3]: Needs to be changed before release?

File: src/services/route/routes.js [hunk 2]
```diff
@@ -117,7 +125,14 @@ const routes = [
         strict: true,
         name: "viewResults",
         component: ViewResults,
-      }
+      },
+      {
+        path: "/error",
```
Comment (Dheeraj-DJP) [inline src/services/route/routes.js:130]: Repeated twice?

**Review by Dheeraj-DJP — CHANGES_REQUESTED** [review]
Few changes
1) There are standalone api calls in few files. Will they need to be wrapped with the error handling?
2) CSS settings for buttons in one file (added in review) needs to wrapped with block, else might affect buttons in other views.

File: src/components/NavBar/User/dropdown/Dropdown.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (sunitakawane) [reply src/components/NavBar/User/dropdown/Dropdown.js:1]: @prantik1998 can you take a look. I do not understand the flow of this code.

File: src/components/NavBar/User/newuser.js [hunk 4]
```diff
@@ -1,192 +1,252 @@
-import React from 'react';
```
[the remark this answers is not in the document]
Comment (sunitakawane) [reply src/components/NavBar/User/newuser.js:1]: @prantik1998

File: src/components/Error/styles.scss [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+.material-icons.red {
+  color: red;
+  font-size: 40px !important;
+}
+.material-icons.reload {
+  font-size: 25px !important;
+  vertical-align: middle;
+}
+
+.btn.btn-primary,
```
[the remark this answers is not in the document]
Comment (sunitakawane) [reply src/components/Error/styles.scss:112]: fixed this

**Review by Dheeraj-DJP — COMMENTED** [review]
Changes looks good

File: src/components/NavBar/User/newuser.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (prantik1998) [reply src/components/NavBar/User/newuser.js:1]: I was not able to import the component

File: src/components/NavBar/User/dropdown/Dropdown.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (prantik1998) [reply src/components/NavBar/User/dropdown/Dropdown.js:1]: Yaa it is repeated because I was unable to import it in the react file

**Review by prantik1998 — COMMENTED** [review]
Approved

**Pull request merged**